src/erldns_app.erl

```diff
@@ -19,37 +19,39 @@
 % Application hooks
 -export([start/2, start_phase/3, stop/1]).
 
+-include_lib("kernel/include/logger.hrl").
+
 start(_Type, _Args) ->
-  lager:debug("Starting erldns application"),
+  ?LOG_DEBUG("Starting erldns application"),
   setup_metrics(),
   erldns_sup:start_link().
 
 start_phase(post_start, _StartType, _PhaseArgs) ->
-  lager:debug("Post start phase for erldns application"),
+  ?LOG_DEBUG("Post start phase for erldns application"),
   erldns_events:add_handler(erldns_event_handler),
 
-  lager:debug("Loading custom zone parsers"),
+  ?LOG_DEBUG("Loading custom zone parsers"),
   case application:get_env(erldns, custom_zone_parsers) of
     {ok, Parsers} -> erldns_zone_parser:register_parsers(Parsers);
     _ -> ok
   end,
 
-  lager:debug("Loading custom zone encoders"),
+  ?LOG_DEBUG("Loading custom zone encoders"),
   case application:get_env(erldns, custom_zone_encoders) of
     {ok, Encoders} -> erldns_zone_encoder:register_encoders(Encoders);
     _ -> ok
   end,
 
-  lager:info("Loading zones from local file"),
+  ?LOG_INFO("Loading zones from local file"),
   erldns_zone_loader:load_zones(),
 
-  lager:info("Notifying servers to start"),
+  ?LOG_INFO("Notifying servers to start"),
   erldns_events:notify(start_servers),
 
   ok.
 
 stop(_State) ->
-  lager:info("Stop erldns application"),
+  ?LOG_INFO("Stop erldns application"),
```

Comment:
Now that we are here, the start of the erldns is at DEBUG level and stop is at INFO level. Maybe we could move start also to INFO level.

Reply:
That will be challenging to convince upstream maintainers to merge such patch.

Reply:
don't we use our own fork?

Reply:
We do, but I almost always push patches to upstream. It take up to a year to get them merge.